astropy/__init__.py

```diff
@@ -324,7 +324,7 @@ def _rebuild_extensions():
 
     if sp.returncode != 0:
         raise OSError('Running setup.py build_ext --inplace failed '
-                      'with error code {0}: try rerunning this command '
+                      'with error code {}: try rerunning this command '
```

Comment:
Is there a specific logic why the latter 3 cases keep the `.format()` syntax while the first one is converted, or does `pyupgrade` simply not handle multi-line statements?

Reply:
Indeed it looks like `pyupgrade` doesn't do multi-line statements.   Grr.

Reply:
https://github.com/asottile/pyupgrade/issues/178

Reply:
it's intentionally timid -- it doesn't want to make your lines longer which might anger flake8